crates/telio-nurse/src/aggregator.rs

```diff
@@ -222,9 +222,15 @@ impl<W: WireGuard> ConnectivityDataAggregator<W> {
         }
     }
 
-    #[allow(dead_code)]
-    pub async fn change_peer_state_direct(
-        &mut self,
+    /// Inform Nurse about direct peer state change.
```

Comment:
I think this is not informing nurse, but recording a change (so that in future nurse can be notified about it).

Reply:
Well, I thought about the whole module, but maybe it is not the best wording ^^'